### PR TITLE
add data format perl scripts

### PR DESCRIPTION
**Pull request by zhangb1**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Adding perl scripts to do the data format before using the Consensus CNV from openPBTA method.
Please see more detail in the Readme.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

I tested locally using the PNOC008 samples. please see the usage and example in the readme

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment: linux,  perl 
* Test files: 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have committed any related changes to the PR

### CODE REVIEW

**Review by yuankunzhu — CHANGES_REQUESTED** [review]
please don't include `.DS_Store` in your commit

File: scripts/data_format/format_control-freeC.pl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        "P_value|P:s" => \$p_value,
+        "INFO|I:s" => \$info,
+		"biospecimens_id|bs:s" => \$bs,
+        "help|h|?" => \$help,
+);
+
+die &usage() if (!defined $p_value || !defined $info || $help);
+
+
+open IF,"$info";
```
Comment (kgaonkar6) [inline scripts/data_format/format_control-freeC.pl:29]: Along gathering the ploidy, do we want to add code snippet to gather purity from the `Sample_Purity` row in *info.txt file as well?

Comment (zhangb1) [reply scripts/data_format/format_control-freeC.pl:29]: @kgaonkar6 Thank you for the comments.`Sample_Purity`  doesn't use in the consensus CNV . does it use somewhere for the analysis, I am not aware of

Comment (kgaonkar6) [reply scripts/data_format/format_control-freeC.pl:29]: We need it for histology bixu responsible columns right?

Comment (zhangb1) [reply scripts/data_format/format_control-freeC.pl:29]: I think the `normal_fraction | tumor_fraction| germline_gender_estimation| tumor_ploidy` in histology file, maybe I miss the Purity one

Comment (kgaonkar6) [reply scripts/data_format/format_control-freeC.pl:29]: ah you are right! The tumor purity is not from controlfreec, it's fractions from Theta2. Sorry!

**Review by kgaonkar6 — APPROVED** [review]
Approving! Looks good to me after the .DS_Store file is removed.

**Review by HuangXiaoyan0106 — APPROVED** [review]
Looks good to me, will approve.

**Pull request merged**